ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -113,6 +121,15 @@
   (validate-bulk-update-post-params json)
   (let [bulk-update-params (json/parse-string json true)
         {:keys [concept-ids]} bulk-update-params
+        concept-ids (if (seq concept-ids)
```

Comment:
I think this could be its own function, and would throw the service errors to avoid having the `_ (when-not` in the `let`

Something like:
```
(defn- filter-concept-ids
  [concept-ids]
  (let [concept-ids (if (seq concept-ids)...])
    (if (seq concept-ids)
      concept-ids
      (errors/throw-service-errors ...))))
```